packages/docregistry/src/mimedocument.ts

```diff
@@ -105,6 +113,7 @@ export class MimeContent extends Widget {
   protected onUpdateRequest(msg: Message): void {
     if (this._context.isReady) {
       this._render();
+      this._fragment = null;
```

Comment:
Any particular reason to null this out (not that I see a problem with it)?

Reply:
`_fragment` value makes sense only until the document is rendered the first time after that value is set. Later the user can scroll the page, for example, and he doesn't want it jumps back to the old position on following rerenderings.